### DISCUSSION TITLE
feat: Switching between relief & shelter locations map views by adding a select element.

### DISCUSSION THREAD

**Discussion by handekaratas96** [acikkaynak/deprem-yardim-frontend#595] [Ideas]
## Task Definition
Disaster map currently displays only victim locations, in addition to this displaying relief/help and shelter locations.

-- Additional Information
** discord username: @Hande Karataş #6520

## Final Notes
https://mui.com/material-ui/react-autocomplete/#checkboxes

**Comment by puskuruk** [comment]
This sound like a whole different project. We might change our base url but use same codebase. Serving this on another URL makes more sense.
P.S. Where do you think you'll be getting this data? AFAIK there is no such data is available in the current API.

**Comment by 0xCakin** [comment]
@usirin would you be able to breakdown this task? It is whole user story and there are many components. Once it is broken down, we can create issues for each team (backend, frontend, data, etc).

**Comment by usirin** [comment]
- [ ] data team should add that what kind of data is (shelter, food, relief effort etc.)
- [ ] backend team will store this data in db.
- [ ] backend will return `id`, `lat`, `lng` in addition to `type` field for `areas` request.
- [ ] backend will return `type` from single data end point.
- [ ] frontend will filter based on this `type` field.
- [ ] frontend we will be filtering with a filter list on right corner of the page.
